docs/source/reference/configuration.rst

```diff
@@ -169,14 +169,17 @@ Restart the mosquitto container for the changes to take effect with the command:
 Web application
 ^^^^^^^^^^^^^^^
 
-Web application configuration provides the ability to customize web components.
+Web application configuration provides the ability to customize web components. All of these are optional.
```

Comment:
Web application configuration provides the ability to customize web components. All of the below directives are optional.